pkgs/development/compilers/gcc/5/default.nix

```diff
@@ -237,10 +237,16 @@ stdenv.mkDerivation ({
 
   # This should kill all the stdinc frameworks that gcc and friends like to
   # insert into default search paths.
-  prePatch = if stdenv.isDarwin then ''
+  prePatch = stdenv.lib.optionalString stdenv.isDarwin ''
```

Comment:
But I guess if we merge this to staging (since we'll already make changes to Linux hash) then we can get rid of the `null`, which would be nice...